count.scm

```diff
@@ -0,0 +1,6 @@
+(define count
+  (lambda (a lat)
+    (cond
+    ((null? lat) 0)
+    (else cons (car lat) (lat cdr))))
```

Comment:
count function compare atom and car lat.
If they are same, add 1 and recur.
How do you compare atom and car lat?
What function do you use?
How do you do add1?
How do you recur the count function?

Reply:
no idea yet